### PR TITLE
[Android] Updated manual for nnstreamer multi-model app v2 @open sesame 06/23 08:00

### PR DESCRIPTION
**Pull request by again4you**
This PR is a succeeding PR of #128

* Apply original patch of @leemgs
* Separate _Android Platform Guide_ from _Example Guide_ since platform guide builds from scratch (i.e building Android full source) so it is only suitable platform developers
* Newly add `arr` (i.e. Android Archive package) usage and its related guide.
* Fixed link errors, typoes, and syntax errors.

**Self evaluation:**
1. Build test: [*]Passed [ ]Failed []Skipped
2. Run test: [*]Passed [ ]Failed []Skipped

### CODE REVIEW

File: android/README.md [hunk 1]
```diff
@@ -1,117 +1,9 @@
...
- * Change the RPATH of executables and libraries
- * Remove declared dependencies on dynamic libraries (DT_NEEDED entries)
-
-If you use another distribution instead of Ubuntu, please visit https://pkgs.org/download/patchelf.
-
-```bash
-sudo apt install patchelf
-man patchelf
-```
+* __Android App Guide using Studio__: If you simply make an Android NNStreamer Application using pre-built `arr` file, then refer to [this guide](https://github.com/nnstreamer/nnstreamer-example/tree/master/android/example_app/README.md).
```
Comment (jaeyun-jung) [inline android/README.md:5]: aar maybe

File: android/example_app/README.md [hunk 2]
```diff
@@ -16,42 +16,48 @@ Also, we assume that you already have experienced Android application developmen
 
 ## Build example
 
-NNStreamer-SSD: Simple object detection example with TF-Lite model
-
-We built a example using GStreamer tutorials and camera2 source for Android.
+We provide various Android examples such as nnstreamer-ssd, nnstreamer-multi, capi-sample and so on. In this guide, we are going to build `NNStreamer-SSD`, which is Simple object detection example with TF-Lite model. We built an example using GStreamer tutorials and camera2 source for Android.
```
Comment (jaeyun-jung) [inline android/example_app/README.md:19]: lower case : Simple

**Comment by again4you** [comment]
### Submit v2
* Fix typo: arr -> aar, Simple -> simple

**Comment by leemgs** [comment]
LGTM

**Comment by leemgs** [comment]
@again4you , Please, fix conflicting files.

File: android/android_platform_guide.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+
+## Build NNstreamer full source with ndk-build
+We recommend that you read an Android NDK manual at https://developer.android.com/studio/build/building-cmdline.
+If you have to compile just C/C++ source code for Android platform, please use 'ndk-build' command.
+Note that You must use 'gradlew' command on the Ubuntu terminal, if you have to compile JNI-based Java source code
+as well as C/C++ JNI source code.  The 'ndk-build' command uses jni/Android.mk by default.
+However, the 'gradlew' command uses the 'build.gradle' file instead of the existing 'jni/Android.mk'.
+
+```bash
+git clone https://github.com/nnsuite/nnstreamer.git
```
Comment (anyj0527) [inline android/android_platform_guide.md:79]: minor typo: nnsuite -> nnstreamer. And how about replace "NNstreamer"s -> "NNStreamer"s in this file?

File: README.md [hunk 4]
```diff
@@ -3,7 +3,7 @@
 This repository shows developers how to create their applications with nnstreamer/gstreamer. We recommend to install nnstreamer by downloading prebuilt binary packages from Launchpad/PPA (Ubuntu) or Download.Tizen.org (Tizen). If you want to build nnstreamer in your system for your example application builds, pdebuild (Ubuntu) with PPA or gbs (Tizen) are recommended for building nnstreamer. This repository has been detached from nnstreamer.git to build examples independently from the nnstreamer source code since Jan-09-2019.
 
 Ubuntu PPA: nnstreamer/ppa [[PPA Main](https://launchpad.net/~nnstreamer/+archive/ubuntu/ppa)]<br />
-Tizen Project: devel:AIC:Tizen:5.0:nnsuite [[OBS Project](https://build.tizen.org/project/show/devel:AIC:Tizen:5.0:nnsuite)] [[RPM Repo](http://download.tizen.org/live/devel%3A/AIC%3A/Tizen%3A/5.0%3A/nnsuite/standard/)]
+Tizen Project: devel:Tizen:6.0:AI [[OBS Project](https://build.tizen.org/project/show/devel:Tizen:6.0:AI)] [[RPM Repo](http://download.tizen.org/live/devel%3A/Tizen%3A/6.0%3A/AI/Tizen_Unified_standard/)]
```
Comment (leemgs) [inline README.md:6]: Impeccable.

**Comment by leemgs** [comment]
LGTM

File: android/android_platform_guide.md (hunk 3 above)
Comment (again4you) [reply android/android_platform_guide.md:79]: Nice Catch! I will.

**Pull request merged**